src/order-validation.ts

```diff
@@ -0,0 +1,148 @@
+import Cart from './cart';
+
+const addOrderValidation = (): void => {
```

Comment:
Would be better either make turn this function to class or decompose on small files with few functions.